### DISCUSSION TITLE
Setting "resourceExposurePolicy" in "Essential Business Headlines" to allow debugging

### DISCUSSION THREAD

**Discussion by GrzegorzLewicz** [microsoft/ALAppExtensions#26265] [General] [answered]
Please change the "allowDebugging" property to true in the manifest file of "Essential Business Headlines"

"resourceExposurePolicy":  {
                               "**allowDebugging":  true,**
                               "allowDownloadingSource":  true,
                               "includeSourceInSymbolFile":  true
}

It will make it easier for partners to find the root cause of some issues. One of our customers (Version: W1 20.4) receives the following error: "Cannot call SetAscending on field Amount because it is not part of the current sorting." in the "RC Headlines Executor" job. The callstack show the issue in "Essential Bus. Headline Mgt."(CodeUnit 1437).TryHandleLargestOrder but when with no possibility to see the code in debugger, it's hard to prepare a fix or workaround.

**Comment by JesperSchulz** [comment, marked as answer]
The "Essential Business Headlines" app is open source. The by far quickest way to get your change processed is to create an issue here on GitHub, wait for it to be approved, and then create a PR with the suggested change. Alternatively, log a BC idea for the product group; however, it might be a long time before your idea gets picked up - pending enough votes for the idea.

**Reply by GrzegorzLewicz** [reply]
Thank you. I've created the issue:
https://github.com/microsoft/ALAppExtensions/issues/26298
I would appreciate if you could process it soon. Thank you in advance.
